modules/deployment/pom.xml

```diff
@@ -16,6 +16,8 @@
 
     <properties>
         <formatterConfigPath>../../formatter.xml</formatterConfigPath>
+        <!-- Mule Connectors and Modules -->
+        <muleJavaModuleVersion>1.3.0-SNAPSHOT</muleJavaModuleVersion>
```

Comment:
`mule` modules should not depend on actual connectors. Options are: Use a test extension or move the test that requires this to integration